public/locales/en/modules/bitcoin.json

```diff
@@ -0,0 +1,20 @@
+{
+    "descriptor": {
+      "name": "Bitcoin",
+      "description": "Displays the current bitcoin information."
+    },
+    "titles":{
+      "recommended-fees":"Recommended Fees",
+      "bitcoin-price":"Bitcoin Price"
+    },
+    "texts":{
```

Comment:
I don't think this is a good name for the category, it's too vague

Reply:
I agree, I just didnt have something to guide me through the coding standards. I used the WeatherWidget as a reference for this whole thing